package.json

```diff
@@ -42,8 +44,9 @@
     ]
   },
   "devDependencies": {
+    "@babel/core": "^7.3.3",
+    "@babel/preset-env": "^7.3.1",
     "babel-plugin-transform-react-jsx": "^6.24.1",
```

Comment:
Should we also update the babel plugin while we're at it?

Reply:
Yea that makes sense. Eventually it might break as `@babel` forks away from the 6.24 apis?